opensrp-app/src/main/java/org/smartregister/repository/TaskRepository.java

```diff
@@ -112,6 +121,18 @@ public static void createTable(SQLiteDatabase database) {
         database.execSQL(CREATE_TASK_PLAN_GROUP_INDEX);
     }
 
+    /**
+     * Migrate the older database by add restriction columns and changing of priority to enum
+     *
+     * @param database the database being upgraded
+     */
+    public static void updatePriorityToEnumAndAddRestrictions(@NonNull SQLiteDatabase database) {
+        DatabaseMigrationUtils.addColumnIfNotExists(database, TASK_TABLE, RESTRICTION_REPEAT, VARCHAR);
```

Comment:
Isn't this an Integer field as declared on line 108?

Reply:
Corrected